src/vic3/tables/triggers.rs

```diff
@@ -924,6 +924,7 @@ const TRIGGER: &[(Scopes, &str, Trigger)] = &[
     (Scopes::Pop, "pop_loyalist_fraction", CompareValue),
     (Scopes::Pop, "pop_neutral_fraction", CompareValue),
     (Scopes::Pop, "pop_radical_fraction", CompareValue),
+    (Scopes::Country.union(Scopes::State), "radical_fraction", CompareValue),
```

Comment:
radical_fraction is already in the table, lower down

Reply:
Aah right, did it in the completely wrong place. 

`	value = radical_fraction`
Is also valid code. I don't know where to add that.

Reply:
There's no place to add that because that's new syntax :) Not supported yet.

The usual place would be the TRIGGER_COMPLEX table, but that expects `(...)` after the trigger name.

Reply:
The problem it's not a trigger, it's an actual value assignment.

Reply:
It might make sense to just add it as a special case in `validate_trigger_key_bv` and `validate_target_ok_this` for now.

Reply:
> The problem it's not a trigger, it's an actual value assignment.

Yeah that would be covered by treating it as a 'target', and triggers with `CompareValue` outputs can be used as targets.

`radical_fraction` is unique in being able to _optionally_ take a `Block`.

Reply:
One solution might be to add a `BlockOrCompareValue` variant to the `Trigger` enum, which checks whether it's getting a `Block` or a `Value`. And update the `trigger_comparevalue` function to treat it as a `CompareValue`.